File: bubbleSort.py
from os import *
from sys import *
from collections import *
from math import *

def bubbleSort(arr,n):
    for i in range(n):
        for j in range(1, n):
            if arr[j] < arr[j - 1]:
                arr[j], arr[j - 1] = arr[j - 1], arr[j]
    pass



# Skip main function if you submit code in coding ninja
if __name__ == "__main__":
    arr = [5, 2, 9, 3, 1]
    n = len(arr)
    bubbleSort(arr,n)

    print("Sorted array:", arr)

File: bubblesort.java

public class bubblesort {
    // Skip main function if you are writing code in coding ninja
    public static void main(String[] args) {
        int[] arr = {5, 2, 9, 3, 1};
        int n = arr.length;

        System.out.print("Original array: ");
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }
        System.out.println();

        bubbleSort(arr, n);

        System.out.print("Sorted array: ");
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }
        System.out.println();
    }

    public static void bubbleSort(int[] arr, int n) {
        for (int i = 0; i < n; i++) {
            for (int j = 1; j < n; j++) {
                if (arr[j] < arr[j - 1]) {
                    int temp = arr[j];
                    arr[j] = arr[j - 1];
                    arr[j - 1] = temp;
                }
            }
        }
    }
}
